template/src/utilities/axiosInstance.js

```diff
@@ -0,0 +1,25 @@
+import axios from 'axios'
+
+const instance = axios.create({
+  baseURL: 'MY_URL',
```

Comment:
Could we get it in env config

Reply:
should create a **service** folder and put it inside

Reply:
done